pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -600,6 +609,29 @@ func (r *ReconcileClusterDeployment) reconcile(request reconcile.Request, cd *hi
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileClusterDeployment) reconcileLogsPVC(cd *hivev1.ClusterDeployment, pvc *corev1.PersistentVolumeClaim, cdLog log.FieldLogger) error {
```

Comment:
This is not a good name for this function. It is not doing any reconciling. It is just creating the PVC if it does not exist.